app/Models/User.php

```diff
@@ -45,4 +45,8 @@ public function setPasswordAttribute($value)
     {
         $this->attributes['password'] = bcrypt($value);
     }
+
+    public function offer_users(){
```

Comment:
Can we make it camel case? because I think it's best practice to make functions names camel case in laravel.